webpack/assets/javascripts/react_app/components/users/PersonalAccessTokens/PersonalAccessTokensSelectors.js

```diff
@@ -1,4 +1,12 @@
+import { STATUS } from '../../../constants';
+import { selectAPIStatus } from '../../../redux/API/APISelectors';
+import { PERSONAL_ACCESS_TOKENS_REQUEST } from './PersonalAccessTokensConstants';
+
 export const selectNewPersonalAccessToken = state =>
   state.personalAccessTokens.newPersonalAccessToken;
 
 export const selectTokens = state => state.personalAccessTokens.tokens;
+
+export const selectIsSubmitting = state =>
+  selectAPIStatus(state, PERSONAL_ACCESS_TOKENS_REQUEST) === STATUS.PENDING ||
```

Comment:
`PERSONAL_ACCESS_TOKENS_REQUEST` is not used in the submit, should be `PERSONAL_ACCESS_TOKEN_FORM_SUBMITTED`